couchbase-sys/src/lib.rs

```diff
@@ -8,3 +8,9 @@
 extern crate openssl_sys;
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
```

Comment:
Was this meant to be here?

Reply:
Indeed, there appears to be a duplication of the extern crate openssl_sys; and include!(concat!(env!("OUT_DIR"), "/bindings.rs")); lines in the code. This duplication seems unintentional and can be safely removed. The primary purpose of this code is to set up attributes, dependencies, and include generated Rust bindings. Thank you for pointing this out, and we will make sure to clean it up in the code.

Reply:
Sorry, I was referring to the commented out unit test below.